addons/mail/models/mail_gateway_allowed.py

```diff
@@ -40,8 +40,11 @@ def get_empty_list_help(self, help_message):
                 Add addresses to the Allowed List
             </p><p>
                 To protect you from spam and reply loops, Odoo automatically blocks emails
-                coming to your gateway past a threshold of <b>%i</b> emails every <b>%i</b>
+                coming to your gateway past a threshold of <b>%(threshold)i</b> emails every <b>%(minutes)i</b>
                 minutes. If there are some addresses from which you need to receive very frequent
                 updates, you can however add them below and Odoo will let them go through.
             </p>
-        ''', LOOP_THRESHOLD, LOOP_MINUTES)
+        ''')  % {
+            'threshold': LOOP_THRESHOLD,
+            'minutes': LOOP_MINUTES,
+        }
```

Comment:
you should keep the percent safe way of having translation parameters:

```suggestion
        ''', threshold=LOOP_THRESHOLD, minutes=LOOP_MINUTES)
```
(using this, if the translation value has an error in %i %(...)i, the original value is used instead instead of getting an error)